assets/src/stories-editor/helpers/index.js

```diff
@@ -1425,6 +1425,19 @@ export const getCallToActionBlock = ( pageClientId ) => {
 	return innerBlocks.find( ( { name } ) => name === 'amp/amp-story-cta' );
 };
 
+/**
+ * Gets the number of megabytes per second for the video.
+ *
+ * @param {Object} media The media object of the video.
+ * @return {number} Number of megabytes per second, or -1 if media details unavailable.
```

Comment:
Why not just return `null` instead in that case? -1 is easy to miss and we don't use such a return type anywhere else.

Reply:
Like so?

```diff
diff --git a/assets/src/stories-editor/blocks/amp-story-page/edit.js b/assets/src/stories-editor/blocks/amp-story-page/edit.js
index e5793fda..167f5a53 100644
--- a/assets/src/stories-editor/blocks/amp-story-page/edit.js
+++ b/assets/src/stories-editor/blocks/amp-story-page/edit.js
@@ -256,7 +256,7 @@ class PageEdit extends Component {
 
 		const colorSettings = this.getOverlayColorSettings();
 		const isExcessiveVideoSize = VIDEO_BACKGROUND_TYPE === mediaType && isVideoSizeExcessive( media );
-		const videoBytesPerSecond = VIDEO_BACKGROUND_TYPE === mediaType ? getVideoBytesPerSecond( media ) : -1;
+		const videoBytesPerSecond = VIDEO_BACKGROUND_TYPE === mediaType ? getVideoBytesPerSecond( media ) : null;
 
 		return (
 			<>
@@ -306,7 +306,7 @@ class PageEdit extends Component {
 										)
 									}
 									{
-										videoBytesPerSecond > 0 && ' ' + sprintf(
+										videoBytesPerSecond && ' ' + sprintf(
 											/* translators: %d: the number of actual megabytes per second */
 											__( 'The selected video is %d MB per second.', 'amp' ),
 											Math.round( videoBytesPerSecond / MEGABYTE_IN_BYTES )
diff --git a/assets/src/stories-editor/helpers/index.js b/assets/src/stories-editor/helpers/index.js
index 7d74cf03..669bc813 100644
--- a/assets/src/stories-editor/helpers/index.js
+++ b/assets/src/stories-editor/helpers/index.js
@@ -1429,11 +1429,11 @@ export const getCallToActionBlock = ( pageClientId ) => {
  * Gets the number of megabytes per second for the video.
  *
  * @param {Object} media The media object of the video.
- * @return {number} Number of megabytes per second, or -1 if media details unavailable.
+ * @return {number|null} Number of megabytes per second, or null if media details unavailable.
  */
 export const getVideoBytesPerSecond = ( media ) => {
 	if ( ! has( media, [ 'media_details', 'filesize' ] ) || ! has( media, [ 'media_details', 'length' ] ) ) {
-		return -1;
+		return null;
 	}
 	return media.media_details.filesize / media.media_details.length;
 };
```